yang/IETF-04/ietf-l3vpn-ntw.yang

```diff
@@ -1406,6 +1406,74 @@ module ietf-l3vpn-ntw {
                            diagnosis purposes. The semantic of the description
                            is local to an implementation.";
                       }
+                      leaf as-override {
+                        type boolean;
+                          default false;
+                          description
+                           "Define whether AS override is enabled, i.e., replace
+                           the AS number of the peer specified in the AS Path
+                           attribute with the local AS number.";
+                        }
+
+                      leaf default-route {
+                        type boolean;
+                        default false;
+                        description
+                         "Define whether default route can be advertised
+                         to its peer.";
+                        }
+
+
+                      container bgp-max-prefix {
+                        leaf max-prefix {
+                          type uint32;
+                          default 5000;
+                        }
+
+                      leaf warning-threshold{
+                        type decimal64 {
+                          fraction-digits 5;
+                          range "0..100";
+                        }
+                          units percent;
+                        default 75;
+                        }
+
+                      leaf violate-action {
+                        type enumeration {
+                          enum warning {
+                            description "Only give warning message when limit is exceeded";
+                          }
+                          enum discard-extra-paths {
+                            description "Discard extra paths when limit is exceeded";
+                          }
+                          enum restart {
+                            description "Restart time interval";
+                          }
+                        }
+                          description
+                           "bpg neighbour max-prefix violate action";
+                       }
+
+                      leaf restart-interval {
+                        type uint16;
+                          description "Time interval (min) after which peering session will be reestablished";
+                          }
+                        }
+
+                      container bgp-timer {
```

Comment:
Add reference and description for the timers. See  draft-ietf-idr-bgp-model-09.
   "RFC 4271, Section 4.4.?
            RFC 4271, Section 10."?;